test/langtools/jdk/jshell/ToolEnablePreviewTest.java

```diff
@@ -25,6 +25,7 @@
  * @test
  * @bug 8199193
  * @summary Tests for the --enable-preview option
+ * @requires vm.continuations
```

Comment:
This seems to be only jshell tests that uses --enable-preview so I think this is okay, just need to update the copyright year.

The update to TEST.ROOT looks okay, just surprised that tests in the langtools tree have never needed to specify requires properties before now.

Reply:
Updated.